.github/workflows/test.yml

```
@@ -3,15 +3,50 @@ name: codacy-coverage-reporter
 on: push
 
 jobs:
-  install-and-test:
-    name: install and test
+  test:
+    name: setup environment and run tests
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
-      - name: npm install
+      - name: Check out repository code
+        uses: actions/checkout@master
+      - name: Node Setup
         uses: actions/setup-node@v2
         with:
           node-version: 14
       - run: npm install
+      - name: Setup Go
+        uses: actions/setup-go@v1
```

```suggestion
        uses: actions/setup-go@v3
```
Let's use [actions/setup-go@v3](https://github.com/actions/setup-go#v3)

This way you will not need to do:
```
      - name: Update PATH
        run: PATH=$(go env GOPATH)/bin:$PATH
```

---

done